sdcm/cluster.py

```diff
@@ -187,6 +188,10 @@ class ScyllaRequirementError(Exception):
     pass
 
 
+class NodeNotReady(Exception):
```

Comment:
We have special module for exceptions here: https://github.com/scylladb/scylla-cluster-tests/blob/master/sdcm/exceptions.py

Reply:
moved it there